client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -343,28 +371,19 @@ const Editor = (props: EditorProps): JSX.Element => {
     editor.addAction({
       id: 'toggle-accessibility',
       label: 'Toggle Accessibility Mode',
-      keybindings: [monaco.KeyMod.CtrlCmd | monaco.KeyCode.F1],
+      keybindings: [monaco.KeyMod.CtrlCmd | monaco.KeyCode.KEY_E],
       run: () => {
-        const currentAccessibility = props.inAccessibilityMode;
-        // The store needs to be updated first, as onDidChangeConfiguration is
-        // called before updateOptions returns
+        const currentAccessibility = storedAccessibilityModes();
+        
+        store.set( 'accessibilityMode', { isAccessibilityModeOn: !currentAccessibility });
         props.setAccessibilityMode(!currentAccessibility);
         editor.updateOptions({
-          accessibilitySupport: currentAccessibility ? 'auto' : 'on'
+          accessibilitySupport: storedAccessibilityModes() ? 'on' : 'off',
```

Comment:
```suggestion
          accessibilitySupport: storedAccessibilityModes() ? 'on' : 'auto',
```
Same argument as above.